src/Compiler/pars.fsy

```diff
@@ -1822,32 +1822,65 @@ classDefnMembersAtLeastOne:
 
 /* The "with get, set" part of a member definition */
 classDefnMemberGetSet: 
-  | WITH classDefnMemberGetSetElements
```

Comment:
I think it would be best to avoid making any grammar changes as part of this.  That should be possible?

Reply:
Perhaps, but I feel like this is a bit of a missed opportunity to separate some logic based on what the parser actually detects.

Having the rule:
```
/* The "get, set" part of a member definition */
classDefnMemberGetSetElements: 
  | classDefnMemberGetSetElement 
     { [$1], None  }
  | classDefnMemberGetSetElement AND classDefnMemberGetSetElement
     { let mAnd = rhs parseState 2
       [$1;$3], Some mAnd }
```

forces the consumer to deal with this in code. Both `get` and `set` or single `get` or single `set` need to be processed correctly where `classDefnMemberGetSetElements` is used.